src/App.js

```diff
@@ -1,19 +1,65 @@
 import React from 'react';
 import './App.css';
 
+import { Form } from './components/Form';
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-}
+const getUserById = userId => (
+  users.find(user => user.id === userId)
+);
+
+const todosWithUsers = todos.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export class App extends React.Component {
+  state = {
+    allTodos: todosWithUsers,
```

Comment:
не добавляй не нужные приставки
```suggestion
    todos: todosWithUsers,
```